apidoc/ostree-sections.txt

```diff
@@ -683,18 +691,6 @@ ostree_repo_finder_override_add_uri
 ostree_repo_finder_override_get_type
 </SECTION>
 
-<SECTION>
-<FILE>ostree-misc-experimental</FILE>
-ostree_repo_get_collection_id
-ostree_repo_set_collection_id
-ostree_validate_collection_id
```

Comment:
I think this should be in the `ostree-core` section and the others should be in the `ostree-repo` section. They're normal APIs and the doc section just didn't get updated in 8fbf19c9f54ba34ae7ebbc540b08566b8dd18848.